.github/workflows/ci.yaml

```diff
@@ -68,16 +67,12 @@ jobs:
         run: make crates
 
       - name: Cache dependencies
-        uses: Swatinem/rust-cache@v1
+        uses: Swatinem/rust-cache@v2
```

Comment:
The `working-directory` key is producing a warning after this update: https://github.com/stm32-rs/stm32-rs/pull/738/files

Looks like it was replaced with the `workspaces` option: https://github.com/Swatinem/rust-cache/blob/master/CHANGELOG.md#200

Reply:
Do you know how to fix (or do you have examples)? I don't understand the syntax.

Reply:
Yeah that is confusing, and non-standard.

I found a working example in another repo: https://github.com/alexschrod/cargo-test-annotations/blob/273b8db69fe01651453551359898c99c1ab2d87a/action.yml

Based on that I think it should probably look like this for v2?

```yaml
      - name: Cache dependencies
        uses: Swatinem/rust-cache@v1
        with:
          workspaces: "${{ matrix.crate }} -> target"
```